### DISCUSSION TITLE
Возможность отмены ежедневного задания

### DISCUSSION THREAD

**Discussion by VladimirOrlovsky** [Sokol-Games-GmbH/PixelCombats.PublicApi#222] [Ideas]
Иногда попадают задания которые не понятны игроку. Допустим я не понял смысла задания "победы на громадных картах". Какие карты относятся к громадным мне не понятно, и в итоге не понятное мне задание нельзя убрать и приходится ждать 24 часа дабы оно убралось.

**Comment by kkohno** [comment]
обдумаем
